code/modules/mob/living/simple_animal/friendly/snake.dm

```diff
@@ -64,9 +72,67 @@
+	friendly_verb_simple = "bite"
+	buffed = FALSE //so that you can't use a wumborian fugu gland on it to allow it to break its oath
+	food_type = list(/obj/item/reagent_containers/food/snacks/grown) //it's vegan (or at least trying to be a vegetarian)
+	careful = TRUE //otherwise, NPC asclepius snakes will kill people by ODing them on omnizine
+	attack_same = TRUE //we "attack" our allies to heal them
+
+/mob/living/simple_animal/hostile/retaliate/poison/snake/asclepius/Initialize()
+	. = ..()
+	var/datum/atom_hud/H = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
+	H.add_hud_to(src)
+	notify_ghosts("A controllable snake of Asclepius has been created in \the [get_area(src)].", source = src, action = NOTIFY_ORBIT, flashwindow = FALSE, header = "Sentient Snake Created")
+
+/mob/living/simple_animal/hostile/retaliate/poison/snake/asclepius/ListTargets(atom/the_target)
+	var/list/patients = list()
+	for(var/mob/living/living_in_oview in oview(vision_range, targets_from)) //get list of things in vision range
+		//can I heal you?
+		if(!living_in_oview.reagents || (living_in_oview.health >= living_in_oview.maxHealth)) //yeah, clone damage, robotic limbs, and such will still cause the snake to bite people it can't fully heal, but eh, even divine doctors can be dentheads sometimes
+			continue
+		if(careful && poison_type.overdose_threshold && living_in_oview.reagents.has_reagent(poison_type, (poison_type.overdose_threshold - poison_per_bite))) //if they're already full of our reagent, let's not remain obsessed with them
+			continue
+		patients += living_in_oview
+	return patients
+
+/mob/living/simple_animal/hostile/retaliate/poison/snake/asclepius/tamed()
+	. = ..()
+	friends.Cut() //we WANT to be able to "attack" our friends so that we can heal them
+
+/mob/living/simple_animal/hostile/retaliate/poison/snake/asclepius/attack_ghost(mob/user)
+	. = ..()
+	if(. || !(GLOB.ghost_role_flags & GHOSTROLE_SPAWNER))
+		return
+	humanize_snek(user)
+
+/mob/living/simple_animal/hostile/retaliate/poison/snake/asclepius/proc/humanize_snek(mob/user)
+	if(key || stat)
+		return
+	var/pod_ask = alert("Become a snake of Asclepius?", "Are you a true pacifist?", "Yes", "No")
+	if(pod_ask == "No" || QDELETED(src))
+		return
+	if(key)
+		to_chat(user, "<span class='warning'>Someone else already took this snake!</span>")
+		return
+	key = user.key
+	log_game("[key_name(src)] took control of [name].")
+
+/mob/living/simple_animal/hostile/retaliate/poison/snake/asclepius/Login()
+	. = ..()
+	to_chat(src, "<span class='boldwarning'>You are one of Asclepius's divine snakes! Unlike a normal snake, when you bite someone, you'll inject them with [poison_per_bite]u of [poison_type.name] instead of toxin(s). Fortunately, due to your medical training, your bites will deal no (direct) damage and you'll never inject anyone with enough [poison_type.name] to overdose them.\n\
+	Above all else, follow the tenets of the Hippocratic Oath.</span>")
```

Comment:
This should probably be handled on the ghost mob policy thing.

Reply:
I don't know how to use that, sorry.

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/POLICYCONFIG.md yote

Reply:
IC flavortext is fine, but if it relies on admin enforcement it needs to be policy-supported (hippocratic oath statement).

Even if this statement was 100% ok, should never turn down an improvement opportunity :)